lib/collectionspace/converter/publicart/organization.rb

```diff
@@ -3,54 +3,79 @@ module Converter
     module PublicArt
       class PublicArtOrganization < Organization
         ::PublicArtOrganization = CollectionSpace::Converter::PublicArt::PublicArtOrganization
+        def redefined_fields
+          @redefined.concat([
+            # not in publicart
+            'contactname',
+            'group',
+            'function',
+            # overridden by publicart
+            'foundingplace'
+          ])
+          super
+        end
+
         def convert
-          run(wrapper: "document") do |xml|
+          run(wrapper: 'document') do |xml|
             xml.send(
-                "ns2:organizations_common",
-                "xmlns:ns2" => "http://collectionspace.org/services/organization",
-                "xmlns:xsi" => "http://www.w3.org/2001/XMLSchema-instance"
+              'ns2:organizations_common',
+              'xmlns:ns2' => 'http://collectionspace.org/services/organization',
+              'xmlns:xsi' => 'http://www.w3.org/2001/XMLSchema-instance'
             ) do
-              # applying namespace breaks import
               xml.parent.namespace = nil
-
-              # TODO: CoreOrganization
-              CSXML.add xml, 'shortIdentifier', config[:identifier]
-
-              CSXML.add_group_list xml, 'orgTerm',
-                                   [{
-                                        "termDisplayName" => attributes["termdisplayname"],
-                                        "mainBodyName" => attributes["mainbodyname"],
-                                    }]
+              PublicArtOrganization.map_common(xml, attributes, config, redefined_fields)
```

Comment:
@Paulmulonzia - This is PublicArtOrganization.map_common, not CoreOrganization.map_common, but we keep these method names consistent, since this method is going to call CoreOrganization.map_common below.